src/Schema/AbstractSchemaManager.php

```diff
@@ -1470,7 +1471,7 @@ protected function _getPortableTableIndexesList($tableIndexes, $tableName = null
                 ];
             }
 
-            $result[$keyName]['columns'][]            = $tableIndex['column_name'];
+            $result[$keyName]['columns'][]            = stripslashes($tableIndex['column_name']);
```

Comment:
Why is this change necessary?

Reply:
Example: 
The expression in functional part of index could be something like
```
(cast(json_extract(`path`,_utf8mb4'$[*]') as char(255) array))
```
So, then you get info from statistic
```
SELECT
     COALESCE(COLUMN_NAME, concat('(', EXPRESSION, ')')) AS Column_Name
FROM information_schema.STATISTICS
```
You've got 
```
(cast(json_extract(`path`,_utf8mb4\'$[*]\') as char(255) array))
```
Quotes will be backslashed.

Reply:
That query of yours is specific to MySQL, but you run the `stripslashes()` on the abstract schema manager. That feels odd.

Reply:
Ok, maybe I could remove backslashes in the SQL Query.. I'll try